### PR TITLE
hard refresh on auth

### PR DESCRIPTION
**Pull request by pablodanswer**
## Description
[Provide a brief description of the changes in this PR]

## How Has This Been Tested?
[Describe the tests you ran to verify your changes]

## Accepted Risk (provide if relevant)
N/A

## Related Issue(s) (provide if relevant)
N/A

## Mental Checklist:
- All of the automated tests pass
- All PR comments are addressed and marked resolved
- If there are migrations, they have been rebased to latest main
- If there are new dependencies, they are added to the requirements
- If there are new environment variables, they are added to all of the deployment methods
- If there are new APIs that don't require auth, they are added to PUBLIC_ENDPOINT_SPECS
- Docker images build and basic functionalities work
- Author has done a final read through of the PR right before merge

## Backporting (check the box to trigger backport action)
Note: You have to check that the action passes, otherwise resolve the conflicts manually and tag the patches.
- [ ] This PR should be backported (make sure to check that the backport attempt succeeds)

### CODE REVIEW

File: web/src/app/auth/login/EmailPasswordForm.tsx [hunk 1]
```diff
@@ -69,9 +66,9 @@ export function EmailPasswordForm({
           if (loginResponse.ok) {
             if (isSignup && shouldVerify) {
               await requestEmailVerification(values.email);
-              router.push("/auth/waiting-on-verification");
+              window.location.href = "/auth/waiting-on-verification";
```
Comment (Weves) [inline web/src/app/auth/login/EmailPasswordForm.tsx:72]: can we add a comment as to why we're doing this?

**Review by Weves — CHANGES_REQUESTED** [review]

File: web/src/app/auth/login/EmailPasswordForm.tsx (hunk 1 above)
Comment (Weves) [reply web/src/app/auth/login/EmailPasswordForm.tsx:72]: it feels "wrong" to do this, so would love to have the justification here to alleviate future confusion

Comment (Weves) [reply web/src/app/auth/login/EmailPasswordForm.tsx:72]: also, would it be possible to just do a router.refresh() or similar?

Comment (pablonyx) [reply web/src/app/auth/login/EmailPasswordForm.tsx:72]: Sadly router.refresh doesn't trigger server side refresh for providers. Could use specific <Link attributes to trigger full refresh, but we'd need to either move our providers client side or have a global refresh we use for this redirect (which feels less maintainable than just triggering global update here).

**Review by Weves — APPROVED** [review]

**Pull request merged**